wis2box-management/wis2box/metadata/station.py

```diff
@@ -345,12 +404,12 @@ def get(ctx, wsi, verbosity):
     results = OrderedDict({
         'station_name': station['station_name'],
         'wigos_station_identifier': station['wigos_station_identifier'],
```

Comment:
get? also for the other keys?

Reply:
I did not write this part of the code, but I believe the station-report from pyoscar always returns the same attributes